### DISCUSSION TITLE
Pages UI redesign

### DISCUSSION THREAD

**Discussion by abkaaar** [abkaaar/allospace#2] [Ideas]
Pages redesign for intuitiveness such as:

- Landing Page
- Space Page
- Footer
- etc

**Comment by looju** [comment]
currently working on the footer

**Reply by abkaaar** [reply]
are you making enhancement and what link would it include?

**Comment by looju** [comment]
I think we definitely need a waiting list feature so we can accumulate enough listings for coworking spaces

**Comment by looju** [comment]
I need the social media links for twitter, LinkedIn, all of them...

**Comment by looju** [comment]
I feel we should replace the features in footer with a news blog. We can use an external api for now to populate the feed

**Reply by abkaaar** [reply]
the footer should be:
Product -> Pricing, Faq.
company -> blog, terms of service , Privacy policy.
social media -> X, Facebook, Instagram.
